doc/source/user/feature-classification/feature-matrix-pool.ini

```diff
@@ -0,0 +1,244 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+[driver.elb]
+title=Enhanced LB Provider
+link=https://docs.otc.t-systems.com/api/elb/elb_zq_0000.html
+
+[driver.vlb]
+title=Dedicated LB Provider
+link=https://docs.otc.t-systems.com/api/elb/elb_zq_0000.html
+
+# Note: These should be in the order they appear in a create call.
+
+[operation.admin_state_up]
+title=admin_state_up
+status=mandatory
+notes=Enables and disables the pool.
+cli=openstack loadbalancer pool create [--enable | --disable] --listener <listener>
+driver.elb=partial
+driver.vlb=partial
+driver-notes.elb=Only active
+driver-notes.vlb=Only active
+
+[operation.alpn_protocol]
+title=alpn_protocol
+status=optional
+notes=List of accepted ALPN protocols (can be set multiple times).
+cli=openstack loadbalancer pool create [--alpn-protocol <protocol>] --listener <listener>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.ca_tls_container_ref]
+title=ca_tls_container_ref
+status=optional
+notes=The reference of the key manager service secret containing a PEM format CA certificate bundle for tls_enabled pools.
+cli=openstack loadbalancer pool create [--ca-tls-container-ref <ca_tls_container_ref>] --listener <listener>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.crl_container_ref]
+title=crl_container_ref
+status=optional
+notes=The reference of the key manager service secret containing a PEM format CA revocation list file for tls_enabled pools.
+cli=openstack loadbalancer pool create [--crl-container-ref <crl_container_ref>] --listener <listener>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.lb_algorithm.LEAST_CONNECTIONS]
+title=lb_algorithm - LEAST_CONNECTIONS
+notes=The pool will direct connections to the member server with the least connections in use.
+cli=openstack loadbalancer pool create --lb-algorithm LEAST_CONNECTIONS --listener <listener>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.lb_algorithm.ROUND_ROBIN]
+title=lb_algorithm - ROUND_ROBIN
+notes=The pool will direct connections to the next member server, one after the other, rotating through the available memeber servers.
+cli=openstack loadbalancer pool create --lb-algorithm ROUND_ROBIN --listener <listener>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.lb_algorithm.SOURCE_IP]
+title=lb_algorithm - SOURCE_IP
+notes=The pool will direct connections to the member server based on a hash of the source IP.
```

Comment:
When the value is SOURCE_IP, the weights of backend servers in the server group are invalid.